package-lock.json

```diff
@@ -1814,7 +1815,7 @@
     "node_modules/@centreon/centreon-frontend": {
       "name": "centreon-frontend",
       "version": "21.10.0",
-      "resolved": "git+https://centreon@github.com/centreon/centreon-frontend.git#3b1322c1b2127593272117d111b6bbdeddea5602",
+      "resolved": "git+https://centreon@github.com/centreon/centreon-frontend.git#32236d536c9a18321bc545fd1624bdad97a67398",
```

Comment:
where branch contains this commit ID please ? 
`32236d536c9a18321bc545fd1624bdad97a67398`

Reply:
In this branch ->
https://github.com/centreon/centreon-frontend/commits/dev-21.10.x

Reply:
this commit https://github.com/centreon/centreon-frontend/commit/3b1322c1b2127593272117d111b6bbdeddea5602